src/batou_ext/ssl.py

```diff
@@ -248,6 +248,7 @@ def update(self):
     -out {{component.fullchain}}
 """
         )
+        self.cmd("setfacl -Rm u:nginx:rX {{componet.key.dir}}")
```

Comment:
Double check https://github.com/flyingcircusio/batou_ext/blob/master/src/batou_ext/acl.py#LL7C22-L7C22 whether useful

Reply:
Unfortunately it only works for files, not folders.